<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>634</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <widget class="QLabel" name="Label_Target_WL">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Target WL (nm)</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QPushButton" name="Button_Update">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="LineEdit_Folders">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLineEdit" name="LineEdit_Target_WL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLineEdit" name="LineEdit_Stop_WL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="11">
    <widget class="QRadioButton" name="radioButton1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>150 L/mm, 4000 nm, 2500-9000 nm</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QComboBox" name="ComboBox_Repetition">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>10 Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100 Hz</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Label_Data_Number">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QComboBox" name="ComboBox_Math_Range">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>50 V^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20 V^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 V^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 V^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 V^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 V^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500 mV^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200 mV^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100 mV^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50 mV^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20 mV^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 mV^2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="Label_Sampling_Number">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Sampling Number</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QPushButton" name="Button_Delete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="10">
    <widget class="QLabel" name="Label_Gratings">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gratings</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="Label_Math_Range">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Math Range</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="Label_Target_WN">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Target WN (cm-1)</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QLabel" name="Label_Sampling_Rate">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Sampling Rate (kS/s)</string>
     </property>
    </widget>
   </item>
   <item row="30" column="0" colspan="14">
    <widget class="QTextEdit" name="textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="Button_Close">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QLabel" name="Label_Stop_WN">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Stop WL (cm-1)</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QPushButton" name="Button_Previous">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="27" column="0">
    <widget class="QLabel" name="Label_Folders_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Folders for plot</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="Label_Repetition">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Repetition</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLineEdit" name="LineEdit_Step_WL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="19" column="2">
    <widget class="QPushButton" name="Button_Measure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Measure</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLineEdit" name="LineEdit_Start_WN">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> background: rgb(220, 220, 220)</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="27" column="1" colspan="4">
    <widget class="QLineEdit" name="LineEdit_Folders2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="Label_Start_WN">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Start WN (cm-1)</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="QPushButton" name="Button_Go">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="LineEdit_Data_Number">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="Label_Start_WL">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Start WL (nm)</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLineEdit" name="LineEdit_Stop_WN">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> background: rgb(220, 220, 220)</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="Label_WL_step">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>WL step (nm)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="2" colspan="12">
    <widget class="QRadioButton" name="radioButton2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>600 L/mm, 1200 nm, 800-3000 nm</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLineEdit" name="LineEdit_Target_WN">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> background: rgb(220, 220, 220)</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QLineEdit" name="LineEdit_Sampling_Number">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> background: rgb(220, 220, 220)</string>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QLineEdit" name="LineEdit_Sampling_Rate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> background: rgb(220, 220, 220)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="Button_Folder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="LineEdit_Start_WL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="Label_Stop_WL">
     <property name="font">
      <font>
       <family>游ゴシック</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Stop WL (nm)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="14" rowspan="31">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="PlotWidget" name="graphicsView1"/>
     </item>
     <item>
      <widget class="PlotWidget" name="graphicsView2"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit2">
       <property name="font">
        <font>
         <family>游ゴシック</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Button_Close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
